zebra-consensus/src/block/check.rs

```diff
@@ -246,7 +260,8 @@ pub fn miner_fees_are_valid(
     // https://zips.z.cash/protocol/protocol.pdf#txnconsensus
     let left = (transparent_value_balance - sapling_value_balance - orchard_value_balance)
         .map_err(|_| SubsidyError::SumOverflow)?;
-    let right = (block_subsidy + block_miner_fees).map_err(|_| SubsidyError::SumOverflow)?;
+    let right = (block_subsidy + block_miner_fees - expected_deferred_amount)
+        .map_err(|_| SubsidyError::SumOverflow)?;
```

Comment:
The comment above `left = ..` should be updated.

Reply:
Updated the comment in https://github.com/ZcashFoundation/zebra/pull/8694/commits/e5eaf38ba10e56a51f02234d80d548a164e0033f.

@daira this likely needs an update to the protocol specification if it's correct.